packages/perseus/src/widgets/radio/base-radio.new.tsx

```diff
@@ -146,9 +124,7 @@ const BaseRadio = function ({
                 // note(matthew): we know this is only getting passed
                 // to a WB Clickable button, so we force it to be of
                 // type HTMLButtonElement
-                // @ts-expect-error - TS2339 - Property 'current' does not exist on type 'never'.
-                const anyNode = ReactDOM.findDOMNode(ref.current) as any;
-                const buttonNode = anyNode as
+                const buttonNode = ReactDOM.findDOMNode(ref.current) as
                     | HTMLButtonElement
```

Comment:
This seemed needlessly redundant. I figured cleaning this up would make it easier to focus on the removal of `findDOMNode` in a future ticket.